disktable.go

```diff
@@ -0,0 +1,650 @@
+/*
```

Comment:
this should prob be "proper" godoc?

Reply:
This one I'm going to punt.  I know that in some circles people seem to think that you can't use /* */ with godoc, but the different between having 100 lines with // and being bracketed by /* */ is simply time savings.  The moment I have to do examples, I'm going straight for /* */.  I don't know of any practical reason why it would cause a problem, plus it is in the language.

The doc comment page on golang.org uses both styles if you look at the "command" example: https://tip.golang.org/doc/comment

Reply:
I have never used `/*` for godocs, TIL thanks